groupstage.js

```diff
@@ -67,6 +73,7 @@ GroupStage.configure({
     // no group size set => league
     o.groupSize = Number(o.groupSize) || np;
     o.meetTwice = Boolean(o.meetTwice);
+    o.numberOfDuels = Number(o.numberOfDuels);
```

Comment:
default 1 maybe?

Reply:
Ok, although meetTwice and numberOfDuels should be the same parameter